dipy/align/scalespace.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, image, num_levels,
             mask = np.asarray(image > 0, dtype=np.int32)
 
         # Normalize input image to [0,1]
-        img = (image - image.min())/(image.max() - image.min())
+        img = (image - np.min(image))/(np.max(image) - np.min(image))
```

Comment:
I don't mind the changes made in this file, but is there a particular reason to make these changes? I believe the code path is identical with this syntax?

Reply:
Changes were made because pytest was having issues with array.min() - see the conversation further up in this pull request. I tried several versions of pytest going back to version 5.0.0, all had the same issue.